Frends.MicrosoftSQL.ExecuteQuery/Frends.MicrosoftSQL.ExecuteQuery.Tests/NonQueryUnitTests.cs

```diff
@@ -130,4 +131,60 @@ public async Task TestExecuteQuery_NonQuery()
             CleanUp();
         }
     }
+
+    [TestMethod]
+    public async Task TestExecuteQuery_DBNullValues()
+    {
+        var options = new Options()
+        {
+            SqlTransactionIsolationLevel = SqlTransactionIsolationLevel.Default,
+            CommandTimeoutSeconds = 30,
+            ThrowErrorOnFailure = true
+        };
+
+        var inputInsert = new Input()
+        {
+            ConnectionString = _connString,
+            Query = $@"INSERT INTO {_tableName} VALUES (1, @Last, @First)",
+            ExecuteType = ExecuteTypes.NonQuery,
+            Parameters = new QueryParameter[]
+            {
+                new() {
+                    Name = "@Last",
+                    Value = null,
+                    SqlDataType = SqlDataTypes.Auto
+                },
+                new() {
+                    Name = "@First",
+                    Value = "Mikki",
+                    SqlDataType = SqlDataTypes.Auto
+                }
+            }
+        };
+
+        var result = await MicrosoftSQL.ExecuteQuery(inputInsert, options, default);
+        Assert.IsTrue(result.Success);
+        Console.WriteLine($"Success {result.Success}");
+
+        var jToken = new JObject(
+            new JProperty("Etu", "Mikki")
+            );
+
+        inputInsert.Parameters = new QueryParameter[]
+            {
+                new() {
+                    Name = "@Last",
+                    Value = jToken["Suku"],
+                    SqlDataType = SqlDataTypes.Auto
+                },
+                new() {
+                    Name = "@First",
+                    Value = jToken["Etu"],
+                    SqlDataType = SqlDataTypes.Auto
+                }
+            };
+
+        result = await MicrosoftSQL.ExecuteQuery(inputInsert, options, default);
+        Assert.IsTrue(result.Success);
```

Comment:
Could you add a third QueryParameter object with a different SqlDataType value, and run a SELECT query to ensure those rows are there and to check how the NULL value is set?